### DISCUSSION TITLE
How can i set up a homepage that looks like {something}.domain.com ? instead of {something}.domain.com/{somethingelse}

### DISCUSSION THREAD

**Discussion by ismaen** [GitbookIO/community#299] [0. General bugs and questions] [answered]
I can't seem to figure out how to set up a custom domain with just sub.domain.com without any slashes after. Anyone know how to do this?

**Comment by sarah-wakefield** [comment, marked as answer]
For a custom domain to be used as-is, it would need to be set for a single **space**.

It's also possible to set a custom domain:

- For a **collection**, in which case the custom domain would be used as-is for the default space in the collection, and the format would be [yourcustomdomain]/v/[spaceURL] for other spaces in the collection.
- For an **organization**, in which case the format would be [customdomain]/[spaceURL] for all spaces owned by that organization.

For reference, we have some more information about this in our documentation, here: [Deciding where to set the custom domain](https://docs.gitbook.com/publishing/custom-domain/location). I hope that helps!

**Reply by ismaen** [reply]
Very helpful, thank you!
